tables/akk.utb

```diff
@@ -0,0 +1,985 @@
+# liblouis: Akkadian
+
+#-index-name: Akkadian
+#-display-name: Akkadian braille
+
+#+language: akk
+#+language: akk-Xsux
```

Comment:
Now I'm not so sure anymore about the "language" metadata. Is this a braille table for the Cuneiform/Akkadian _script_, or the Akkadian _language_?